packages/nativescript-audio-recorder/index.ios.ts

```diff
@@ -275,4 +278,33 @@ export class AudioRecorder extends Observable implements IAudioRecorder {
       });
     });
   }
+
+  /**
+   * Notify events by name and optionally pass data
+   */
+  public _sendEvent(eventName: string, data?: any) {
+    this.notify(<any>{
+      eventName,
+      object: this,
+      data: data,
+    });
+  }
+  /**
+   * Events
+   */
+  public static startedEvent = 'startedEvent';
+  public static stoppedEvent = 'stoppedEvent';
+  public static completeEvent = 'completeEvent'; //will pass the recording filename
+  public static errorEvent = 'errorEvent'; //will pass the error object
```

Comment:
these consts are declared several times. Is there a reason that they can't be initialized on the common file and declared in the d.ts file?